website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -84,29 +84,29 @@ The React Native documentation must be updated to reflect each addition or chang
 
 We want to thank all the contributors mentioned below who have submitted and merged pull requests as well as those who have reviewed and commented on issues.
 
-### Merged Pull Request
```

Comment:
@lunaleaps I reformatted how this list and how the fixes list was formatted to highlight the authors of the PRs first and then the PRs.